package.json

```diff
@@ -6,19 +6,24 @@
     "start": "gatsby develop",
     "develop": "yarn start",
     "build": "gatsby build",
-    "serve": "gatsby serve"
+    "serve": "gatsby serve",
+    "lint": "eslint src/"
   },
   "dependencies": {
     "axios": "^0.17.1",
+    "babel-plugin-emotion": "^9.2.10",
     "babel-plugin-transform-runtime": "^6.23.0",
     "d.js": "^0.7.5",
+    "emotion": "^9.2.10",
+    "eslint-plugin-react": "^7.11.1",
     "gatsby": "2.0.7",
     "gatsby-plugin-react-helmet": "3.0.0",
     "makerule": "^1.0.5",
     "prop-types": "^15.6.2",
     "react": "^16.5.2",
     "react-dom": "^16.5.2",
+    "react-emotion": "^9.2.10",
```

Comment:
Is there value sticking within the Gatsby ecosystem?
https://www.gatsbyjs.org/packages/gatsby-plugin-emotion/

Reply:
e.g do we lose any Gatsby functionality by Emotion being outside the build tools?

Reply:
Ok I will take a look, definitely didn't know about this.